index.js

```diff
@@ -1,67 +1,99 @@
 #!/usr/bin/env node
 
-import { fileURLToPath } from 'node:url'
 import { join } from 'node:path'
 import { fork } from 'node:child_process'
+import { fileURLToPath } from 'node:url'
+import { createReadStream } from 'node:fs'
+import { createInterface } from 'node:readline'
+
 import variationsStream from 'variations-stream'
+
+import Constants from './constants.js'
 import ArgsParser from './argsParser.js'
 import JWTValidator from './jwtValidator.js'
-import Constants from './constants.js'
-
-const __dirname = fileURLToPath(new URL('.',
-  import.meta.url))
 
-const args = new ArgsParser()
+const __dirname = fileURLToPath(new URL('.', import.meta.url))
+const numberFormatter = Intl.NumberFormat('en', { notation: 'compact' }).format
```

Comment:
This is the magic  used for displaying speed like `875K/s`, `1M/s`, `1.1M/s`

Reply:
Nice trick! :) i was not aware you could simply use `Intl`!